File: Solution/每日一题/2024.9.8/8.go
func clearDigits(s string) string {
    var res []byte
    for _, c := range s {
        if unicode.IsDigit(c) {
            res = res[:len(res)-1]
        } else {
            res = append(res, byte(c))
        }
    }
    return string(res)
}

File: Solution/每日一题/2024.9.8/8.cs
public class Solution {
    public string ClearDigits(string s) {
        StringBuilder res = new StringBuilder();
        foreach (char c in s.ToCharArray()) {
            if (Char.IsDigit(c)) {
                res.Remove(res.Length - 1, 1);
            } else {
                res.Append(c);
            }
        }
        return res.ToString();
    }
}

